src/card/collapsible/_collapsibleCard.scss

```diff
@@ -0,0 +1,9 @@
+.card__content {
+  overflow: hidden;
+
+  transition: height 0.5s ease;
+}
+
+.card__content--collapsed {
+  visibility: hidden;
+}

```

Comment:
Do we need this lines?
```suggestion

```

Reply:
Without setting visibility hidden, collapsed content can still be seen by search functions but it can not locate the text. I'm not sure if it can be seen by text-to-speech readers too.
![image](https://user-images.githubusercontent.com/42540003/125622167-96ad5529-a609-4e3e-9d47-921c17a3cd4b.png)